src/force-graph.js

```diff
@@ -405,10 +407,11 @@ export default Kapsule({
       }
     }, false);
 
+
     // Handle click events on nodes/links
     container.addEventListener('click', ev => {
       if (state.hoverObj) {
-        state[`on${state.hoverObj.type}Click`](state.hoverObj.d);
+        state[`on${state.hoverObj.type}Click`](state.hoverObj.d, ev);
       } else {
         state.onBackgroundClick();
```

Comment:
The event object could be included also in background clicks, for consistency.

Reply:
Agree 👍